web/src/lib/utils/helpers.tsx

```diff
@@ -310,3 +310,11 @@ export const makeButtonIcon = (svgFilename: string, size = "20px"): ReactElement
     alt=""
   />
 );
+
+export const isIndustryIdGeneric = (industry: Industry) => {
+  return industry.id === "generic";
+};
```

Comment:
I think this should be moved to the industry file in shared, so it's available in any project.

Reply:
I'll make this update